src/sentry/api/event_search.py

```diff
@@ -635,11 +635,17 @@ def _handle_basic_filter(self, search_key, operator, search_value):
         # If a date or numeric key gets down to the basic filter, then it means
         # that the value wasn't in a valid format, so raise here.
         if search_key.name in self.date_keys:
-            raise InvalidSearchQuery(f"{search_key.name}: Invalid format for date field")
+            raise InvalidSearchQuery(
+                f"{search_key.name}: Invalid format for date field. Expected ISO 8601 or +/-duration."
```

Comment:
Instead of making a user look up ISO8601, why not include an example? It'd also let the user copy and paste the example to fill in the field as well. We also already do this for an error message:
https://github.com/getsentry/sentry/blob/443efcca22e69723935923fcd0250ad30b546bd7/src/sentry/api/event_search.py#L1590

Reply:
Hm. I thought about that but thought it made the error message too long. Additionally, not all event_search usages allow the user to copy and paste. For example, the performance landing page only shows a toast which disappears when clicked on.

I think we can shorten "Invalid format for date field" to "Invalid date" though, which gives us more room for an example.